services/data_plane_manager/src/main/resources/application.properties

```diff
@@ -13,3 +13,10 @@ grpc.threads-pool-name = grpc-thread-pool
 #logging.level.root=INFO
 
 
+mq.type=pulsar
+#####Pulsar configuration#####
+pulsar.url=pulsar://127.0.0.1:6650
+pulsar.unicast.topic=unicast-topic1
+host.ip.to.group.topic.map=group-topic1:192.168.131.131,10.10.10.11 group-topic2:192.168.131.131,11.11.11.12
```

Comment:
This is an OK short-term solution. Let us plan a long-term solution. One possibility to store the mapping in node metadata manager, as this is a host-level resource.

@chenpiaoping @VanderChen what do you think of?

Reply:
Create a tracking issue https://github.com/futurewei-cloud/alcor/issues/490

@VanderChen If you can take this item, that would be great. Let us sync on Slack for more details.